### ISSUE TITLE
poor inference results

### ISSUE THREAD

**Issue by manuel-88** [TimoSaemann/ENet#14]
Thank you for publishing your code. But why does the algorithm work much better on the cityscape test set images than on random traffic images from the internet? Is there some preprocessing necessary?

**Comment by TimoSaemann** [comment]
Since the net was trained with images of the size 1024x512 px, the input images should have this size. However the results are worse because Deep Neural Networks generally assume that the input
data distribution between the training and test data are similar. As the input data distribution changes, due to domain shift, the performance of the network degrades.

**Comment by manuel-88** [comment]
What is the best approach to adapt the distribution of the test images to the train data distribution?

**Comment by TimoSaemann** [comment]
That's a good question. This is a field associated with Domain Adaptation. It is out of the scope of my comment to answer this with great depth, but in general much research is still needed to improve the generalisation of deep neural networks.

**Comment by manuel-88** [comment]
Okay. Thank you for your help.
